py-polars/polars/internals/frame.py

```diff
@@ -2194,7 +2194,7 @@ def join(
         ╰──────┴──────┴─────┴───────╯
 
         Asof join
-        =========
+        ---------
```

Comment:
Also with `----`, sphinx-build complains.

Reply:
Yes, the Asof Join is not recognized by sphinx. I think we must use somehting else for this? The sphinx way of creating a bold text or something like htat.